src/main/java/org/broadinstitute/consent/http/service/VoteService.java

```diff
@@ -362,22 +365,36 @@ protected void notifyCustodiansOfApprovedDatasets(List<Dataset> datasets, User r
           custodianMap.get(submitter).add(d);
         }
       }
-      // Data Custodians and Data Depositor
-      List<String> custodianEmails = d.getProperties()
-          .stream()
-          .filter(p ->
-              (Objects.nonNull(p.getSchemaProperty()) && p.getSchemaProperty()
-                  .equalsIgnoreCase("dataCustodianEmail")) ||
-                  p.getPropertyName().equalsIgnoreCase("Data Depositor"))
-          .map(DatasetProperty::getPropertyValueAsString)
-          .distinct()
-          .toList();
+
+      // Data Custodians
+      List<String> custodianEmails = d.getDataCustodianEmails();
       if (!custodianEmails.isEmpty()) {
         userDAO.findUsersByEmailList(custodianEmails).forEach(u -> {
           custodianMap.putIfAbsent(u, new HashSet<>());
           custodianMap.get(u).add(d);
         });
       }
+
+      // Data Depositors
+      List<String> depositors = d.getDataDepositors();
+      List<String> depositorEmails = new ArrayList<>();
+      String emailRegex = "^[a-zA-Z0-9_+&*-]+(?:\\."+
+          "[a-zA-Z0-9_+&*-]+)*@" +
+          "(?:[a-zA-Z0-9-]+\\.)+[a-z" +
+          "A-Z]{2,7}$";
```

Comment:
I am usually hesitant to implement our own regexes for this kind of thing. It's usually better to let a common library do this for you. I recommend using https://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/EmailValidator.html for this. You can find the library here: https://mvnrepository.com/artifact/commons-validator/commons-validator/1.7 and add it to the `pom.xml` file. Implementation code might look something like this:
```
EmailValidator validator = EmailValidator.getInstance();
List<String> validEmails = depositors.stream().filter(e -> validator.isValid(e)).toList();
```